packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -1960,33 +1960,25 @@ class InteractiveGraph extends React.Component<Props, State> {
     }
 
     render() {
-        // Mafs shim
-        const mafsFlags = this.props.apiOptions?.flags?.["mafs"];
-        if (shouldUseMafs(mafsFlags, this.props.graph)) {
-            const box = getInteractiveBoxFromSizeClass(
-                this.props.containerSizeClass,
-            );
-            const gridStep =
-                this.props.gridStep ||
-                Util.getGridStep(this.props.range, this.props.step, box[0]);
-            const snapStep =
-                this.props.snapStep || Util.snapStepFromGridStep(gridStep);
+        const box = getInteractiveBoxFromSizeClass(
+            this.props.containerSizeClass,
+        );
+        const gridStep =
+            this.props.gridStep ||
+            Util.getGridStep(this.props.range, this.props.step, box[0]);
+        const snapStep =
+            this.props.snapStep || Util.snapStepFromGridStep(gridStep);
 
-            return (
-                <StatefulMafsGraph
-                    {...this.props}
-                    flags={this.props.apiOptions?.flags}
-                    ref={this.mafsRef}
-                    gridStep={gridStep}
-                    snapStep={snapStep}
-                    box={box}
-                    showTooltips={!!this.props.showTooltips}
-                    readOnly={this.props.apiOptions?.readOnly}
-                />
-            );
-        }
         return (
-            <LegacyInteractiveGraph ref={this.legacyGraphRef} {...this.props} />
```

Comment:
Will the `LegacyInteractiveGraph` component be removed in a future PR?

Reply:
Yup, it's a separate task!